docs/chapter1/further-develop-app/dev-admin-proxy.md

```diff
@@ -52,6 +54,13 @@ When your app is built and deployed to production, it will not have a webpack de
 
 To make dev mode more similar to production, Electrode offers a reverse proxy server that combines webpack dev server and your app server under a single port.
 
+### Up front check list
+
```

Comment:
To set up your reverse proxy server, you must perform these tasks.

1. [Enable Reverse Proxy](#enable-reverse-proxy): Specify the env variable `APP_SERVER_PORT` to run the app node server on a different port.
2. [Persist APP_SERVER_PORT](#persist-app_server_port): Update the app server to listen on the port from `APP_SERVER_PORT`.

Reply:
Changes committed